otdfctl/docs/man/policy/dynamic-value-mappings/create.md

```
@@ -0,0 +1,83 @@
+
+For more information about subject condition sets, see the `subject-condition-sets` subcommand.
+
+## Examples
+
+Create a dynamic value mapping entitling 'read' where a patient assignment matches the requested value:
+```shell
+otdfctl policy dynamic-value-mappings create --attribute 891cfe85-b381-4f85-9699-5f7dbfe2a9ab --selector '.patientAssignments[]' --operator IN --action read
+```
+
+Create a dynamic value mapping scoped by Attribute Definition FQN with a substring operator:
+```shell
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)